edgesite/cmd/edgesite/app/server.go

```diff
@@ -61,8 +61,7 @@ runs on edge nodes and manages containerized applications.`,
 			gis.Start(beehiveContext.Done())
 			registerModules(config)
 			// start all modules
-			core.StartModules()
-			core.GracefulShutdown()
+			core.Run()
```

Comment:
It might be clearer to split it into two calls? I'm not sure.

Reply:
The `Run` method combines two methods and has full comment

Reply:
Looks good.